src/lib/db/seed.ts

```diff
@@ -143,6 +144,15 @@ export const seed = async (db: Database, much = false): Promise<void> => {
 		await createDefaultEntities(toMainAccountServiceRequest, spaces, nextActor, repos);
 	}
 
+	const adminHub = await repos.hub.loadAdminHub();
+	await UpdateHubService.perform({
+		...toMainAccountServiceRequest(),
+		params: {
+			actor: mainActorCreator.actor_id,
+			hub_id: ADMIN_HUB_ID,
+			settings: {...adminHub!.settings, instance: {minimumPasswordLength: 3}},
+		},
+	});
```

Comment:
Two things here - it was clobbering any existing `settings.instance` value, and the passwords we seed are 1 character, I don't see a reason to restrict this in dev.

```suggestion
	const adminHub = await repos.hub.loadAdminHub()!;
	await UpdateHubService.perform({
		...toMainAccountServiceRequest(),
		params: {
			actor: mainActorCreator.actor_id,
			hub_id: ADMIN_HUB_ID,
			settings: {
				...adminHub.settings,
				instance: {...adminHub.settings.instance, minimumPasswordLength: 1},
			},
		},
	});
```